### PR TITLE
AF-1916 - Fix Docks Tooltip

### PR DESCRIPTION
**Pull request by RishiRajAnand**
AF-1916 Fix Docks Tooltip

- Replaced old implementation (button sliding out on hover) with the Tooltip on hover implementation. 
- Added two test cases to test Tooltip title and position.

### CODE REVIEW

**Comment by kie-ci8** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by pefernan** [comment]
ok to test

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by pefernan** [comment]
@ederign @ohonchar could you please take a look?

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by ohonchar** [comment]
> @ederign @ohonchar could you please take a look?

thnx, took for testing

File: uberfire-showcase/uberfire-webapp/hs_err_pid1653.log [hunk 1]
```diff
@@ -0,0 +1,390 @@
+#
+# There is insufficient memory for the Java Runtime Environment to continue.
+# Native memory allocation (mmap) failed to map 1431306240 bytes for committing reserved memory.
+# Possible reasons:
+#   The system is out of physical RAM or swap space
+#   The process is running with CompressedOops enabled, and the Java Heap may be blocking the growth of the native heap
```
Comment (pefernan) [inline uberfire-showcase/uberfire-webapp/hs_err_pid1653.log]: This file should be removed!

Comment (RishiRajAnand) [reply uberfire-showcase/uberfire-webapp/hs_err_pid1653.log]: Thanks @pefernan .Sure, I'll remove the file and commit the changes.

**Comment by ohonchar** [comment]
@RishiRajAnand good work, now it is meets the requirements. Testing finished.
@pefernan is this `Refresh` related to this ticket? Screenshot_20190415_215326

File: uberfire-showcase/uberfire-webapp/hs_err_pid1653.log (hunk 1 above)
Comment (RishiRajAnand) [reply uberfire-showcase/uberfire-webapp/hs_err_pid1653.log]: The file has been deleted and merged with previous commit.

**Comment by pefernan** [comment]
@ohonchar sorry but tooltip is not on the scope of this PR. Here we are only tackling the side docks

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by ederign** [comment]
@RishiRajAnand congrats on your first collaboration to the team!!!!! Sorry for the delay on to merge this PR! :)

**Comment by ederign** [comment]
Let's now wait for the jenkins!

**Pull request merged**